orchestra/actions/install.py

```diff
@@ -255,8 +281,8 @@ def _binary_archive_repo_name(self):
         if self.component.binary_archives:
             binary_archive_repo_name = self.component.binary_archives
             if binary_archive_repo_name not in self.config.binary_archives_remotes.keys():
-                raise Exception(f"The {self.component.name} component wants to push to an unknown binary-archives repository ({binary_archive_repo_name})")
-            return binary_archive_repo_name
```

Comment:
You dropped a `return`! Gesù!
Fixed in 5351b9cb7d3a222aaa52106146da079f2ee440b0.